src/pages/EmergencyViewPage/EmergencyViewPage.scss

```
@@ -20,25 +21,25 @@ $tab-item-room-width: 180px;
   flex-direction: column;
   z-index: z(emergency-content);
   height: 100%;
+  position: absolute;
+  height: calc(100vh - #{$reserved-page-height});
+  width: 100%;
 
-  &__banner {
-    position: absolute;
-    height: calc(100vh - #{$navbar-height});
-    width: 100%;
-    background-size: cover;
-    background-repeat: no-repeat;
-  }
-
-  &_main {
-    margin-bottom: $spacing--xxl;
+  &__main {
+    margin-bottom: $spacing--xxl * 2;
+    margin-top: -($spacing--xxl * 2);
+    padding: $spacing--xxl * 2 $tab-item-height 0px $tab-item-height;
+    background-image: $bg-image;
```

A version with the background firmly attached and stretched. This still will not resolve the issue of the window being made scrollable, so it might be a good idea to get the background on a higher element, like `.EmergencyView` itself.

```suggestion
  &__main {
    margin-bottom: $spacing--xxl * 2;
    margin-top: -($spacing--xxl * 2);
    padding: $spacing--xxl * 2 $tab-item-height 0px $tab-item-height;
    background-image: $bg-image;
    background-attachment: fixed;
    background-size: cover;
    background-position: top left;
```

---

fixed c8355dc